Force setup.py to use distutils in order to break setuptools cyclic dependency.

--- appdirs-1.4.0/setup.py
+++ appdirs-1.4.0/setup.py
@@ -2,7 +2,7 @@
 import sys
 import os
 import os.path
-from setuptools import setup
+from distutils.core import setup
 import appdirs
 
 tests_require = []
@@ -38,8 +38,6 @@
         Programming Language :: Python :: 3.2
         Topic :: Software Development :: Libraries :: Python Modules
         """.split('\n') if c.strip()],
-    test_suite='test.test_api',
-    tests_require=tests_require,
     keywords='application directory log cache user',
     author='Trent Mick',
     author_email='trentm@gmail.com',
